amoro-ams/amoro-ams-dashboard/pom.xml

```diff
@@ -113,4 +113,22 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
+        <profile>
+            <id>skip-webui-build</id>
+            <build>
+                <pluginManagement>
+                    <plugins>
+                        <plugin>
+                            <groupId>com.github.eirslett</groupId>
+                            <artifactId>frontend-maven-plugin</artifactId>
+                            <configuration>
+                                <skip>true</skip>
```

Comment:
I think this configuration should be turned off, **false** by default.

Reply:
hi @czy006, this profile will not affect the default value.

https://github.com/apache/flink/blob/b72ce532fdd77bca86a030be1b9878be4c097546/flink-runtime-web/pom.xml#L113